src/components/AboutPage/AboutPage.scss

```diff
@@ -47,14 +49,14 @@ $bg-primary: #59bdee;
 }
 
 @media only screen and (min-width:1277px) {
-    .about-text {
-        color: white;
-        font-size: 23px;
-    }
+    // .about-text {
+    //     color: white;
+    //     font-size: 23px;
+    // }
 
-    .text-center {
-        text-align: center;
-    }
+    // .text-center {
+    //     text-align: center;
+    // }
 }
```

Comment:
Do we need this? If not, let's remove these comments. Once they are removed, I can merge it to the main branch. Thanks.

Reply:
Great catch! Thanks I will address.